pkg/mount/mount.go

```diff
@@ -2,11 +2,16 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// The mount package implements functions for mounting and unmounting
-// file systems and defines the mount interface.
+// Package mount implements mounting, moving, and unmounting file systems.
 package mount
 
+// Mounter is an object that can be mounted.
 type Mounter interface {
-	Mount() error
+	// Mount mounts the file system at path.
+	Mount(path string, flags uintptr) error
```

Comment:
nice! Abstracts the thing that serves blocks from where and how you mount it. 
One question I just realized: when you talk to a 9p server for example, you can say 9p2000 or 9p2000.whatever. So that comes with the mount system call. So linux in some cases has mount "data" that is BOTH about the thing serving the blocks and how you mount that thing -- per-mount data can extend per-device data. Should we have the opts here too and concatenate the per-device mount opts with the per-mount mountopts?

Reply:
I think which protocol still goes with "thing that serves blocks" and not "where and how you mount it". I can imagine some cases where you'd still need per-mount and per-device data, but for now let's keep it as is until we need it?